contracts/interfaces/ICover.sol

```diff
@@ -37,6 +37,7 @@ enum CoverUintParams {
 struct PoolAllocationRequest {
   uint40 poolId;
   uint coverAmountInAsset;
+  uint allocationId;
```

Comment:
`PoolAllocationRequest` is user input. If the pool is new - it should be max uint, if the pool is not new - it should be the previous allocation id (which we already know). As a consequence it doesn't seem like we need any user input here. Remove this field.